core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -385,5 +385,10 @@ oppia.controller('SettingsTab', [
     $scope.closeRolesForm = function() {
       $scope.isRolesFormOpen = false;
     };
+
+    // Meathod to check if title is given
```

Comment:
Typo - Method. Actually, I don't think this comment is needed, you can remove it.

Reply:
I removed the comment, I think the function name was enough.